crates/criterion_compat/criterion-0.5.1/src/analysis/mod.rs

```diff
@@ -256,6 +256,59 @@ pub(crate) fn common<M: Measurement, T: ?Sized>(
             );
         }
     }
+
+    if criterion.should_save_baseline() && std::env::var("CODSPEED_ENV").is_ok() {
+        codspeed::collect_walltime_results(id, criterion, &iters, avg_times);
+    }
+}
+
+mod codspeed {
+    use crate::{measurement::Measurement, report::BenchmarkId, Criterion};
+
+    pub fn create_uri_and_name<M: Measurement>(
```

Comment:
Could we add a docstring callout to make sure we keep this in sync with how instrumented mode generates uri ?

I believe it happens [here](https://github.com/CodSpeedHQ/codspeed-rust/blob/e4bd2bc7a6f228f7877e989c6a9e4996110aeb9e/crates/criterion_compat/src/compat/group.rs#L70-L75)